pages/conference/2021.tsx

```diff
@@ -53,6 +147,78 @@ export default function TwentyTwentyOne() {
           <Anchor href="mailto:desiree@nativesintech.org">desiree@nativesintech.org</Anchor>.
         </p>
       </section>
+      <section className="section">
+      <h2 className="h2">Agenda</h2>
+        <h3 className="h3">Friday, November 5th — Language Revitalization Workshops</h3>
+        {/* <img src="https://pbs.twimg.com/media/FDNF7pmWQAM5q2_?format=jpg&name=medium" style={{height: "400px", margin: "auto", marginTop: "30px"}} /> */}
+        {/* <h4 className="h4">Details</h4> */}
+        <ul className="ul" style={{listStyle: "none", marginTop: "15px"}}>
+          <li className="li">Workshop 1: <strong>1:00pm - 3:00pm CDT</strong></li>
+          <li className="li">Workshop 2: <strong>6:00pm - 8:00pm CDT</strong></li>
```

Comment:
The font color for <strong> appears to be set because on my dark theme I can see "Workshop 1:" in white, but "1:00pm - 3:00pm CDT" is black/gray.

Reply:
I just pushed another tweak to this, replacing <strong> with <span style={{fontWeight: 600}}>, which should hopefully be okay from a color perspective!